src/main/java/com/sarality/db/query/SimpleQueryBuilder.java

```diff
@@ -46,6 +48,20 @@ public SimpleQueryBuilder withFilter(QueryClause clauseBuilder) {
     return this;
   }
 
+  //TODO (Satya) make the measureTypes an enum
+  public SimpleQueryBuilder withAggregate(String measureType, Column measureColumn) {
+    String columnName = measureType + "(" + measureColumn.getName() + ") AS " + measureType + "_" + measureColumn
```

Comment:
We should be doing this here. The only thing we should do is keep track of the functions that we apply to each column. Keep a list of Enum (or String for now) as the columnFunctionList